### ISSUE TITLE
Events for custom smartzoom-element children get lost

### ISSUE THREAD

**Issue by bocksebastian** [e-smartdev/smartJQueryZoom#34]
Events which are bound to child-elements of my smartzoom-element get lost after initialisation.
This is caused by the call of jQuery.remove() in line 94. It removes all assosiated data and events from the element and all the children. 

So to improve the library for compatibility with other use cases it is recommended to use jQuery.detach() instead to keep all assosiated events and data after removing from DOM and appending in line 99.
